docs/get-details/transactions/index.md

```diff
@@ -675,19 +675,56 @@ Here are three example scenarios and how the round range may be calculated for e
+You can also manually set a **flat fee**. If you choose this method, make sure that your fee covers at least the [minimum transaction fee (`min-fee`)](../../rest-apis/algod/v2#transactionparams), which can be obtained from the suggested parameters method call in each of the SDKs.  Flat fees may be useful for applications that want to guarantee showing a specific rounded fee to users or for a transaction that is meant to be sent in the future where the network traffic conditions are unknown.
+
+# Pooled transaction fees
+
+The Algorand protocol supports pooled fees, where one transaction can pay the fees of other transactions within the same atomic group. 
+
+For atomic transactions, the fees set on all transactions in the group are summed. This sum is compared against the protocol determined expected fee for the group and may proceed as long as the sum of the fees is at least the required fee for the group.
+
+<center>![Atomic Pooled Fees](../imgs/atomic_transfers-3.png)</center>
+<center>*Atomic Pooled Fees*</center>
+
+!!! note
+    [Inner transactions](/get-details/dapps/smart-contracts/apps/#inner-transactions) may have their fees covered by the outer transactions but they may not cover outer transaction fees.
+
+!!! info
+    Full running code examples for each SDK and both API versions are available within the GitHub repo at [/examples/atomic_transfers](https://github.com/algorand/docs/tree/master/examples/atomic_transfers) and for [download](https://github.com/algorand/docs/blob/master/examples/atomic_transfers/atomic_transfers.zip?raw=true) (.zip).
+
+An example of setting a pooled fee on a group of two transactions in javascript:
+
+```js
+const suggestedParams = await client.getTransactionParams().do()
+suggestedParams.flatFee = true;
+suggestedParams.fee = 2 * algosdk.ALGORAND_MIN_TX_FEE;
+```
+
+Here we're directly setting the fee to be 2x the min fee since we want to cover both transactions. 
 
 # Setting First and Last Valid
 
-Unless you have specific security concerns or logical constraints embedded within a specific Algorand Smart Contract, it is generally recommended that you set your default range to the maximum, currently 1000. This will give you an ample window of validity time to submit your transaction. 
+Unless you have specific security concerns or logical constraints embedded within a specific Algorand Smart Contract, it is generally recommended that you set your default range to the maximum, currently 1000. This will give you an ample window of validity time to submit your transaction. 
```

Comment:
Actually it's not completely clear it's always the best choice.
For example, if you want to use the min possible fee and want to retry in case of congestion, it is best to use a small window, because retrying before the end of the window is quite dangerous